pyccel/codegen/printing/cwrappercode.py

```diff
@@ -915,7 +915,7 @@ def _print_Module(self, expr):
             static_funcs = expr.funcs
         function_signatures = '\n'.join('{};'.format(self.function_signature(f)) for f in static_funcs)
 
-        interfaces = expr.interfaces
+        interfaces = list(expr.interfaces)
         interface_funcs = [f.name for i in interfaces for f in i.functions]
         funcs = interfaces + [f for f in expr.funcs if f.name not in interface_funcs]
```

Comment:
I don't see why we should create a new list from `expr.interfaces`, which is iterable already:

```python
interfaces = expr.interfaces  # not needed, renaming is just for clarity
interface_funcs = [f.name for i in interfaces for f in i.functions]
funcs = [*interfaces, *(f for f in expr.funcs if f.name not in interface_funcs)]
```

Reply:
You are right, I hadn't considered this syntax

Reply:
Fixed in ad837cff

Reply:
80339504